Gems/ScriptCanvasPhysics/Code/CMakeLists.txt

```diff
@@ -19,9 +19,9 @@ ly_add_target(
             Legacy::CryCommon
         PUBLIC
             Gem::ScriptCanvas.Extensions
-    AUTOGEN_RULES
-        *.ScriptCanvasFunction.xml,ScriptCanvasFunctionRegistry_Header.jinja,AutoGenFunctionRegistry.generated.h
-        *.ScriptCanvasFunction.xml,ScriptCanvasFunctionRegistry_Source.jinja,AutoGenFunctionRegistry.generated.cpp
+     AUTOGEN_RULES
```

Comment:
This is no longer aligned anymore
```suggestion
    AUTOGEN_RULES
```

Reply:
Still needed, this module has ScriptCanvas functions that need to use these rules

Reply:
I meant the indentation is not aligned.

Reply:
I must've been tired and read "This is no longer needed" instead of "aligned".